carma_cooperative_perception/src/multiple_object_tracker_component.cpp

```diff
@@ -577,7 +581,12 @@ auto MultipleObjectTrackerNode::execute_pipeline() -> void
     const auto clusters{mot::cluster_detections(detections_, 0.75)};
     for (const auto & cluster : clusters) {
       const auto detection{std::cbegin(cluster.get_detections())->second};
-      track_manager_.add_tentative_track(std::visit(make_track_visitor, detection));
+      const auto uuid_str{mot::get_uuid(detection).value()};
+      const mot::Uuid new_uuid{
+        std::to_string(lifetime_generated_track_count_++) +
+        uuid_str.substr(std::size(uuid_str) - 4, 4)};
```

Comment:
Let's add a comment here why it is 4. I believe it is solely because CARLA's unique ids are in 3 digits and we need 1 more for the "-" character right? Same in the place below.

Reply:
Done